### DISCUSSION TITLE
Reverse Encoder Direction?

### DISCUSSION THREAD

**Discussion by TheLegendOfMart** [Jyers/Marlin#449] [Q&A] [answered]
I have a Voxelab Aquila which is basically an Ender 3 V2. It uses the same control board and I'm guessing display but the rotary encoder is backwards.

I tried compiling my own with all three reverse encoder options enabled but it didn't do anything.

I realise this is just for the Ender 3 V2 but any help would be appreciated.

**Comment by Jyers** [comment, marked as answer]
If you know what you're doing you could swap some stuff around in "Marlin\src\lcd\extui\lib\creality_dwin\rotary_encoder.cpp" If you need help with what to change let me know.
